TheMovieDb/Utils/Extensions/UIViewExtensions.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  UIViewExtensions.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 30/10/21.
+//
+
+import UIKit.UIView
+
+extension UIView {
```

Comment:
This goes against YAGNI principle, just have the methods you will use

Reply:
I deleted functions that I dont use